packages/core/src/dataTypes/boolean/finalizeControl.ts

```diff
@@ -2,12 +2,15 @@ import type { PartialBooleanControlType } from "./types"
 
 export function finalizeControl(
   props: Omit<PartialBooleanControlType<boolean>, "allowNullInfer">,
-): Required<Omit<PartialBooleanControlType<boolean>, "allowNullInfer">> {
+): Required<
+  Omit<PartialBooleanControlType<boolean>, "allowNullInfer" | "displayName">
```

Comment:
This actually can be removed now that I've changed the `displayName` definition in `PartialControlType` to `string | null`. When it was undefined, the `finalizeControl` fns for all data types were complaining because of the `Required<>` typing. As work around I omitted the `displayName` property from being required.